python/py-jwt/Portfile

```diff
@@ -14,15 +14,13 @@ license             MIT
 maintainers         {petr @petrrr} openmaintainer
 
 description         JSON Web Token implementation in Python
-long_description    ${description}
+long_description    {*}${description}
 
 homepage            https://github.com/jpadilla/pyjwt
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
-distname            ${python.rootname}-${version}
 
-checksums           rmd160  bfe7a0ea07327593b728ba82766d809ae4c1be75 \
-                    sha256  8d59a976fb773f3e6a39c85636357c4f0e242707394cadadd9814f5cbaa20e96 \
-                    size    41979
+checksums           rmd160  3aaf0f2fe2ed6f6faed1ec9bd3900baff27bb1ae \
+                    sha256  fba44e7898bbca160a2b2b501f492824fc8382485d3a6f11ba5d0c1937ce6130 \
+                    size    60092
 
 python.versions     27 35 36 37 38 39
```

Comment:
the version you are updating to support Python 3.6+ so you'll need to deal with the PY27/PY35 subports: either by removing them here and all recursive dependencies, or pinning them to the latest supported version

Reply:
also, it doesn't appear to use `pytest-runner` anymore, so that should probably be removed from `depends_build`